app/server.go

```diff
@@ -18,16 +23,56 @@ func logging(f http.HandlerFunc) http.HandlerFunc {
 	}
 }
 
+func getCustomTLSConfig(caFile string) (*tls.Config, error) {
+	tlsConfig := new(tls.Config)
+	certs, err := ioutil.ReadFile(caFile)
+
+	if err != nil {
+		return tlsConfig, err
+	}
+
+	tlsConfig.RootCAs = x509.NewCertPool()
+	ok := tlsConfig.RootCAs.AppendCertsFromPEM(certs)
+
+	if !ok {
+		return tlsConfig, errors.New("Failed parsing pem file")
+	}
+
+	return tlsConfig, nil
+}
+
 func main() {
-	connectionString := fmt.Sprintf(
-		"mongodb://%s:%s",
-		os.Getenv("PESTCONTROL_DB_IP"),
+	b := new(strings.Builder)
+
+	fmt.Fprint(b, "mongodb://")
+	if user := os.Getenv("PESTCONTROL_DB_USER"); user != "" {
+		fmt.Fprintf(b, "%s:%s@", user, os.Getenv("PESTCONTROL_DB_PW"))
+	}
+	fmt.Fprintf(
+		b,
+		"%s:%s",
+		os.Getenv("PESTCONTROL_DB_HOST"),
 		os.Getenv("PESTCONTROL_DB_PORT"),
 	)
-	db, err := models.NewDB(connectionString)
+
+	var tlsConfig *tls.Config
+
+	if caFilePath := os.Getenv("DOCDB_CERT_PATH"); caFilePath != "" {
```

Comment:
So is this environment variable only going to be set when/if we use DocumentDB?

Reply:
Yeah pretty much it just needs to be set to `rds-combined-ca-bundle.pem` if we want to use it because the way the docker image is made already downloads it and puts it in the image.